src/elements/alt-style/AltStyleElementForm.tsx

```diff
@@ -21,22 +21,20 @@ import { keyTakeawaysFields } from "./AltStyleElementSpec";
 export const AltStyleElementTestId = "AltStyleElement";
 
 const RepeaterChild = styled(Body)`
-  padding: 8px 8px 16px 8px;
-  &:not(:last-child) {
-    border-bottom: 1px dashed ${neutral[60]};
+  &:first-child {
+    margin-top: ${space[3]}px;
   }
+  margin-bottom: ${space[3]}px;
   position: relative;
 `;
 
 const RepeatedFieldsWrapper = styled("div")`
   width: 100%;
-  padding: 0 8px;
```

Comment:
will this affect the recipes use case?

Reply:
Shouldn't do as this form isn't used by the recipe element